containers/fingerbank-db/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine
+
+ARG FINGERBANK_BUILD_API_KEY
+
+RUN apk update && apk add curl
+
+RUN mkdir -p /usr/local/fingerbank/db
+
+RUN curl --fail -H "Authorization: Bearer ${FINGERBANK_BUILD_API_KEY}" https://api.fingerbank.org/api/v2/download/db > /usr/local/fingerbank/db/fingerbank_Upstream.db
```

Comment:
I suggest to add a retry option like we have during build of Fingerbank perl client:

https://github.com/fingerbank/perl-client/blob/e1ae4b2d6aacf8c90c18b1f3e82ec8dfc449cfe6/Makefile#L135-L137